packages/api-kintone/src/invoice/updateInvoices.ts

```diff
@@ -1,5 +1,5 @@
 import { RecordID, Revision } from '@kintone/rest-api-client/lib/src/client/types';
-import { updateAllRecords } from '../common/updateAllRecords';
+import { updateAllRecords } from '../common/updateRecords';
```

Comment:
```suggestion
import { updateRecords} from '../common/updateRecords';
```

社用スマホで見ていて、わかりませんが。多分これになるん？w

Reply:
失礼しました。
コンパイルエラーも解消しましたので、ご確認をお願いします。